addons/service.autoexec/README.md

```diff
@@ -0,0 +1,3 @@
+# service.autoexec
+
+A service add-on which allows or disallows the execution of autoexec.py
```

Comment:
Would make sense to add a paragraph on where autoexec.py is to be located. I.e.:

For systems without profiles configured the file should be located in the root of the `userdata` folder. In the case of profiles it should be located in the root `userdata/profilename`.